apps/web/package.json

```diff
@@ -170,7 +170,7 @@
     "module-alias": "^2.2.2",
     "msw": "^0.42.3",
     "postcss": "^8.4.18",
-    "tailwindcss": "^3.2.1",
+    "tailwindcss": "3.3.1",
```

Comment:
i think we should use "^3.3.1" here.

Reply:
agree

Reply:
> i think we should use "^3.3.1" here.

Thanks, @Udit-takkar I didn't notice it when I update the package.